.github/workflows/helm-multi-release.yaml

```diff
@@ -0,0 +1,200 @@
+name: Helm Multi Release
+# Reusable workflow for packaging and publishing Helm charts having multiple subcharts on GitHub pages
+
+on:
+  workflow_call:
+    inputs:
+      charts-dir:
+        description: "The directory containing the Helm chart and `.bumpversion.cfg` file."
+        required: true
```

Comment:
Can you please order inputs so that required ones are on top?

Reply:
This is not resolved yet as `gh-pages-branch` is a required input, but is below `artifact-dir`, which is not required. Please sort every input that has `required: true` to the top and every input that has `required: false` to the bottom